src/sentry/newsletter/base.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import
+
+
+class Newsletter(object):
+    PRODUCT_CAMPAIGN = 1
```

Comment:
In what way would this be more flexible? On the other side, it's store in the database as an integer so we can just say, "campaign 1 == product email". It's just an ENUM so we're not slinging random strings around the internet without much meaning.

But I can change it if you want. We'd just need to then come up with what this should be since I h ave no idea. At least with it just being `1`, we can define this meaning later when we actually have more than 1.

To me `campaign` is a finite set of things. It's directly related to the email list they're subscribed to, and we wouldn't be having a ton of different things here. Maybe you're mixing this up with `referrer`, which is an arbitrary string that is less structured.